bin/desi_fit_circles

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python
+
+import sys
+from astropy.table import Table
+import numpy as np
+from scipy import optimize
+import matplotlib.pyplot as plt
+import argparse
+
+# from desimeter.transform.xy2qs import xy2qs, qs2xy
+from desimodel.focalplane import xy2qs, qs2xy
+
+#- Transform between CS5 x,y and curved focal surface
+def xy2uv(x, y):
+    q, s = xy2qs(x, y)
+    qrad = np.radians(q)
+    u = s*np.cos(qrad)
+    v = s*np.sin(qrad)
+    return u, v
+
+def uv2xy(u, v):
+    s = np.sqrt(u**2 + v**2)
+    qrad = np.arctan2(v, u)
+    q = np.degrees(qrad)
+    x, y = qs2xy(q, s)
+    return x, y
```

Comment:
Let's put all these blat2foo transforms into the transforms library instead of in this script.  I'm sure we will want to reuse them.  Please add docstrings since this is being promoted from single use exploratory code to the desimeter library.